### PR TITLE
doc(standards): add defect standards

### PR DESCRIPTION
**Pull request by Nintendot**
## Overview

Put in defect reporting and tracking standards (around JIRA's `TD Defect` and DOMO)

### Features

- create RA/testing/standards/defect documentation

#### Meta

- [x] provide a descriptive topic
- [x] provide an overview of contribution
- [x] no sensitive content included, such as:
  - security & privacy policy violating content
  - content considered competitive intelligence
  - keys, tokens or credentials
- [x] documentation format follows [this template][template]
- [x] fork is up to date ([see this guide from github](https://help.github.com/articles/syncing-a-fork/))
- [x] "work in progress" commits are squashed ([see "Squashing Commits"](https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History))
- [x] commits follow the [Karma][karma-format] or [Commitizen](https://www.npmjs.com/package/commitizen) format

[template]: ../.template.md
[karma-format]: https://karma-runner.github.io/1.0/dev/git-commit-msg.html

### CODE REVIEW

File: testing/standards/defect.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## What
+
+We need a standard process of how to
+- Properly report defects
+- Properly monitor defects
+
+To accomplish the above, we use [JIRA][JIRA] for issue tracking, and align on using the issue type `TD Defect` for all production and staging defect reports.
+
+This specific issue type contains useful fields for diagnosis, it also enables data collection and visualization through domo, so that we can do data analytis on defects across all teams.
```
Comment (crimeminister) [inline testing/standards/defect.md]: s/analytis/analysis/

File: testing/standards/defect.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## How
+
+JIRA:
+
+Make sure the issue type `TD Defect` is enabled for your project by following this [google doc][Enable TD Defect]
+
+When logging defects, make sure to provide as much information as possible, such as:
+
+- Defect Priority
```
Comment (crimeminister) [inline testing/standards/defect.md]: Does it make sense to enumerate the fields and values of `TD Defect`?

File: testing/standards/defect.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+A: That’s because it’s not configured for your project, refer to the documentation: [enabling your project for TD Defect][Enable TD Defect] for instructions.
+
+
+## Who
+
+@qa
+
+## Reference
+* [How to enable your JIRA project to use TD Defect](https://drive.google.com/open?id=1RnsCCp0q7grCmi2nRqOB0-Cnhd1mMBox7uhUSg1EB2k)
+* [TD Defect initialtive doc](https://docs.google.com/document/d/1kupC2NN8nfqjnRILcpYNbeiDbMOye9ngK0T8hL3wqw8)
```
Comment (crimeminister) [inline testing/standards/defect.md]: s/initialtive/initiative/

File: testing/standards/defect.md (hunk 1 above)
Comment (Nintendot) [reply testing/standards/defect.md]: updated, ty!

File: testing/standards/defect.md (hunk 2 above)
Comment (Nintendot) [reply testing/standards/defect.md]: You are right, I changed it to just use 3 examples instead

File: testing/standards/defect.md (hunk 3 above)
Comment (Nintendot) [reply testing/standards/defect.md]: updated, ty! (I need a spellchecker in VS code :/)

**Pull request merged**